onyo/lib/commands.py

```diff
@@ -5,6 +5,7 @@
 import logging
 from typing import Dict, Iterable, Optional
 from pathlib import Path
+from collections.abc import Iterable as iter
```

Comment:
I don't understand this one.
1. `typing.Iterable` is an alias for `collection.abc.Iterable`.
2. `as iter` would shadow the builtin `iter`. That's a non-obvious thing to deal with.
3. The usage I see (that `isinstance` test), isn't really what we'd want there. (See comment there)